apps/rewards/app/components/Content/MyRewards.js

```diff
@@ -163,33 +120,22 @@ const renderRecurringDividend = (reward, amountTokens) => {
     </Text>
   )
   const disbursementDate = (disbursements[claims] || disbursements[claims-1]).toDateString()
-  const status = claims === disbursements.length ? <Status type="claimed" /> : (
-    Date.now() > disbursements[claims].getTime() ? <Status type="ready" /> :
-      <Status type="pending" />
-  )
-  return [ description, disbursementDate, status, displayAmount ]
+  const Status = getStatus(reward)
+  return [ description, disbursementDate, Status, displayAmount ]
```

Comment:
Did `status` need to be capitalized in these places?

Reply:
Not really